### ISSUE TITLE
Perl scripts renaming has not been updated in subsequent scripts

### ISSUE THREAD

**Issue by lewismc** [joshua-decoder/joshua#244]
A very recent commit cf. https://github.com/joshua-decoder/joshua/commit/a61f290d27b3b9d12338040a07d9da834c36d017 renamed a number of files e.g.
``` 
...s/training/penn-treebank-detokenizer.perl → scripts/preparation/detokenize.pl
File renamed without changes.
0 scripts/lowercase.perl → scripts/preparation/lowercase.pl
File renamed without changes.
0 scripts/training/normalize-punctuation.pl → scripts/preparation/normalize.pl
File renamed without changes.
0 ...pts/training/penn-treebank-tokenizer.perl → scripts/preparation/tokenize.pl
```
It looks like not all scripts were updated with these changes meaning that invocation of pipelines fails very early on. This is easy fixed and I will submit a PR in 2 minutes.

**Comment by mjpost** [comment]
I think I fixed this already, may have forgotten to push, will check in 30

matt (from hand computer)

**Comment by lewismc** [comment]
I am getting the following when running
/bin/bash: /work/03702/tg830018/wrangler/joshua/scripts/lowercase.pl: No
such file or directory
Thanks Matt I am also having a look at it.

On Wed, Feb 24, 2016 at 4:50 PM, Matt Post <notifications@github.com> wrote:

> I think I fixed this already, may have forgotten to push, will check in 30
>
> matt (from hand computer)
>
> > On Feb 24, 2016, at 7:37 PM, Lewis John McGibbney <
> notifications@github.com> wrote:
> >
> > A very recent commit cf. a61f290 renamed a number of files e.g.
> >
> > ...s/training/penn-treebank-detokenizer.perl → scripts/preparation/
> detokenize.pl
> > File renamed without changes.
> > 0 scripts/lowercase.perl → scripts/preparation/lowercase.pl
> > File renamed without changes.
> > 0 scripts/training/normalize-punctuation.pl → scripts/preparation/
> normalize.pl
> > File renamed without changes.
> > 0 ...pts/training/penn-treebank-tokenizer.perl → scripts/preparation/
> tokenize.pl
> > It looks like not all scripts were updated with these changes meaning
> that invocation of pipelines fails very early on. This is easy fixed and I
> will submit a PR in 2 minutes.
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/244#issuecomment-188538421>
> .
>



-- 
*Lewis*

**Comment by lewismc** [comment]
PR sent to https://github.com/joshua-decoder/joshua/pull/245
This is working sweet as a nut.
Thanks

On Wed, Feb 24, 2016 at 4:53 PM, Lewis John Mcgibbney <
lewis.mcgibbney@gmail.com> wrote:

> I am getting the following when running
> /bin/bash: /work/03702/tg830018/wrangler/joshua/scripts/lowercase.pl: No
> such file or directory
> Thanks Matt I am also having a look at it.
>
> On Wed, Feb 24, 2016 at 4:50 PM, Matt Post <notifications@github.com>
> wrote:
>
>> I think I fixed this already, may have forgotten to push, will check in 30
>>
>> matt (from hand computer)
>>
>> > On Feb 24, 2016, at 7:37 PM, Lewis John McGibbney <
>> notifications@github.com> wrote:
>> >
>> > A very recent commit cf. a61f290 renamed a number of files e.g.
>> >
>> > ...s/training/penn-treebank-detokenizer.perl → scripts/preparation/
>> detokenize.pl
>> > File renamed without changes.
>> > 0 scripts/lowercase.perl → scripts/preparation/lowercase.pl
>> > File renamed without changes.
>> > 0 scripts/training/normalize-punctuation.pl → scripts/preparation/
>> normalize.pl
>> > File renamed without changes.
>> > 0 ...pts/training/penn-treebank-tokenizer.perl → scripts/preparation/
>> tokenize.pl
>> > It looks like not all scripts were updated with these changes meaning
>> that invocation of pipelines fails very early on. This is easy fixed and I
>> will submit a PR in 2 minutes.
>> >
>> > —
>> > Reply to this email directly or view it on GitHub.
>> >
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/joshua-decoder/joshua/issues/244#issuecomment-188538421>
>> .
>>
>
>
>
> --
> *Lewis*
>



-- 
*Lewis*

**Comment by lewismc** [comment]
There is one last big here, please see the following
https://github.com/joshua-decoder/joshua/search?utf8=%E2%9C%93&q=normalize-punctuation.pl
There is mention of the file normalize-punctuation.pl but this file does not exist.

**Comment by mjpost** [comment]
Fixed with commit e6091aa836e613030540350415f11059048d0122.

**Comment by lewismc** [comment]
Fantastic thanks @mjpost
